src/getHumanAge.test.js

```diff
@@ -7,4 +7,36 @@ describe('getHumanAge', () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
+
+  test('should discard the remainder', () => {
+    expect(getHumanAge(27, 27)).toEqual([2, 2]);
+  });
+
+  test('should return [0, 0] for catAge = 0, dogAge = 0', () => {
+    expect(getHumanAge(0, 0)).toEqual([0, 0]);
+  });
+
+  test('should return [0, 0] for catAge = 14, dogAge = 14', () => {
```

Comment:
The test case is inconsistent with the task rules. If catAge = 14 and dogAge = 14, it's unlikely the human years would be 0 for both. Confirm the conversion rules.